WordPress/Classes/Utility/App Configuration/AuthenticationHandler.swift

```diff
@@ -4,4 +4,12 @@ protocol AuthenticationHandler {
     func shouldPresentUsernamePasswordController(for siteInfo: WordPressComSiteInfo?, onCompletion: @escaping (WordPressAuthenticatorResult) -> Void)
 
     func presentLoginEpilogue(in navigationController: UINavigationController, for credentials: AuthenticatorCredentials, onDismiss: @escaping () -> Void) -> Bool
+
+    // WPAuthenticator style overrides
+    var statusBarStyle: UIStatusBarStyle { get }
```

Comment:
I'm not sure these 100% feel like they belong in here, but I don't have much of a better suggestion 🤷‍♂️

Reply:
I agree, and I didn't have one either. I can always move them in the future.